### PR TITLE
Re-enable invokers docker_container module

### PR DESCRIPTION
**Pull request by kpavel**
## Description

- bumped ansible version to 2.5.0 where userns support added
- updated Readme files accordingly
- modified invoker deploy.yml to support docker_container module

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [x] Invoker
- [x] Deployment
- [x] General tooling
- [x] Documentation

## Types of changes

- [x] Enhancement or new feature (adds new functionality).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [x] I updated the documentation where necessary.

### CODE REVIEW

**Comment by markusthoemmes** [comment]
Ha, @cbickel and I just talked about this this morning. Thanks a lot for doing this 🎉

File: ansible/roles/invoker/tasks/deploy.yml [hunk 1]
```diff
@@ -135,95 +135,110 @@
...
+    ports_to_expose: ["{{ invoker.port + groups['invokers'].index(inventory_hostname) }}:8080"]
+
+- name: expose additional ports if jmxremote is enabled
+  when: jmx.enabled
+  set_fact:
+    ports_to_expose: "{{ ports_to_expose }} + [ \"{{ jmx.basePortInvoker + groups['invokers'].index(inventory_hostname) }}:{{ jmx.basePortInvoker + groups['invokers'].index(inventory_hostname) }}\" ] + [ \"{{ jmx.rmiBasePortInvoker + groups['invokers'].index(inventory_hostname) }}:{{ jmx.rmiBasePortInvoker + groups['invokers'].index(inventory_hostname) }}\" ]"
+
+- name: prepare invoker env
+  set_fact:
+    env: {
```
Comment (dubee) [inline ansible/roles/invoker/tasks/deploy.yml]: Are the curl braces necessary here?

Comment (kpavel) [reply ansible/roles/invoker/tasks/deploy.yml]: done

**Comment by chetanmeh** [comment]
@kpavel For some of my work I need support for #3477 and was looking for way to extend the environment variables passed. I believe approach taken in this PR (of building env first and then passing it to docker module) is the way forward for #3477 also. 

Any thoughts if such a requirement can be addressed as part of this PR?. Or I can wait for this one to be merged and build up on that

**Comment by chetanmeh** [comment]
@kpavel Just checking if you are still working on this PR?

**Comment by kpavel** [comment]
@chetanmeh I'm waiting for input from @dubee as i'm not sure what is blocking it from to be merged There new conflicts which i will resolve once i have input.

**Pull request merged**